wit-definitions/worker/wit/worker.wit

```diff
@@ -0,0 +1,119 @@
+package wavs:worker@0.5.0-alpha.11;
+use wavs:types/chain@0.5.0-alpha.11 as chain-types;
+use wavs:types/core@0.5.0-alpha.11 as core-types;
+use wavs:types/service@0.5.0-alpha.11 as service-types;
+
+use wasi:io/poll@0.2.0;
+use wasi:clocks/monotonic-clock@0.2.0;
+use wasi:io/error@0.2.0;
+use wasi:io/streams@0.2.0;
+use wasi:http/types@0.2.0 as http-types;
```

Comment:
This confuses me a little. You removed wasi-http-0.2.0 but we can still reference it here?
Also, were the other deps just not used?

Reply:
working on cleaning this up, you're right, there's some unused stuff here too